docs/asciidoc/modules/ROOT/pages/background-operations/triggers.adoc

```diff
@@ -136,23 +136,24 @@ The `$removedNodeProperties` parameter has the same structure and logic (of cour
 Same thing regarding `assignedRelationshipProperties` and `removedRelationshipProperties`,
 with the only difference that instead of `node: NODE(n)` key, we'll have `relationship: RELATIONSHIP(n)`.
 
-For example, if we want to create a trigger that at every set, update 2 property "time" and "lasts" 
+For example, if we want to create a trigger that at every set, update 2 property "time" and "lasts"
 with the current date and the property updated, we can do:
 
 [source,cypher]
 ----
 CALL apoc.trigger.add('setLastUpdate',
-'UNWIND keys($assignedNodeProperties) as k 
-UNWIND $assignedNodeProperties[k] AS map
-with map.node as node, collect(map.key) as propList
-match (n) where id(n) = id(node)
-where not "lasts" in propList               // to prevent loops
-SET n.time = date(),  n.lasts = propList', {phase:'afterAsync'});
+  "UNWIND keys($assignedNodeProperties) AS k
+  UNWIND $assignedNodeProperties[k] AS map
+  WITH map.node AS node, collect(map.key) AS propList
+  MATCH (n)
+  WHERE id(n) = id(node) AND NOT 'lasts' in propList // to prevent loops
+  SET n.time = date(),  n.lasts = propList",
+  {phase: 'afterAsync'});
 ----
 
 In the example above, we put `match (n) where id(n) = id(node)` to demonstrate that the we pull the node by id into parameters.
 Anyway, we can get rid off this one and change last row with `SET node.time = date(),  node.lasts = propList`.
-Note that we have to put the clause `where not "lasts" in propList` to prevent infinite cascade SET.
+Note that we have to add the condition `and not "lasts" in propList` to prevent infinite cascade SET.
```

Comment:
We could be even more specific about how this would work:
```suggestion
Note that we have to add the condition `and not "lasts" in propList` to prevent an infinite loop as this `SET` will trigger this query again.
```